packages/edit-site/src/components/page-pages/index.js

```diff
@@ -117,6 +121,18 @@ export default function PagePages() {
 					postStatuses[ page.status ] ?? page.status,
 				enableSorting: false,
 			},
+			{
+				header: 'Date',
+				id: 'date',
+				cell: ( props ) => {
+					const formattedDate = dateI18n(
+						getSettings().formats.datetimeAbbreviated,
+						getDate( props.row.original.date )
+					);
+					return <time>{ formattedDate }</time>;
+				},
+				enableSorting: false,
```

Comment:
This field is used to sort the table by default. Why a user shouldn't be able to?

Reply:
Maybe a user can do it in the future, but it might be more convoluted when we have composite fields or something that would need to provide the sorting args. I think more of the APIs will evolve/created per iteration and use case.

Reply:
Follow-up in https://github.com/WordPress/gutenberg/pull/55388